### PR TITLE
Raw Images: minor update

### PR DESCRIPTION
**Pull request by demoulinv**
Remove applyToneCurve parameter in imageProcessing
cameraInit: Test empty dcpDatabase only if dcp is required

### CODE REVIEW

File: src/software/utils/main_imageProcessing.cpp [hunk 1]
```diff
@@ -801,7 +797,6 @@ int aliceVision_main(int argc, char * argv[])
                 options.rawColorInterpretation = rawColorInterpretation;
             }
             options.colorProfileFileName = view.getColorProfileFileName();
-            options.applyToneCurve = applyToneCurve;
```
Comment (fabiencastan) [inline src/software/utils/main_imageProcessing.cpp:804]: Should it not also be removed from ImageReadOptions?

**Comment by fabiencastan** [comment]
There is no reason to have the abc file modified in this PR.

File: src/software/utils/main_imageProcessing.cpp (hunk 1 above)
Comment (demoulinv) [reply src/software/utils/main_imageProcessing.cpp:804]: Done

**Review by fabiencastan — APPROVED** [review]

**Comment by fabiencastan** [comment]
Arf, there was still the unjustified modification to the ABC file in the PR!

**Pull request merged**